### PR TITLE
fix online failures in helio

### PR DESCRIPTION
**Pull request by nabobalis**

### CODE REVIEW

File: sunpy/net/helio/hec.py [hunk 1]
```diff
@@ -40,6 +40,13 @@ def votable_handler(xml_table):
     fake_file = io.BytesIO()
     fake_file.write(xml_table)
     votable = parse_single_table(fake_file)
+    # This is a workaround: in astropy 4.1 they made votable parse
+    # char type to strings over bytes which has changed the type of our return
+    # The fix here is burning this module down.
```
Comment (Cadair) [inline sunpy/net/helio/hec.py:45]: The fix on master is to make the rest of the module expect str.

```suggestion
```

File: sunpy/net/helio/hec.py [hunk 2]
```diff
@@ -174,14 +181,14 @@ def select_table(self):
...
         while True:
             user_input = input(f"\nPlease enter a table number between 1 and {len(table_list)} "
                                "('e' to exit): ")
             if user_input.lower() == "e" or user_input.lower() == "exit":
                 return None
             if user_input.isdigit() and 1 <= int(user_input) <= len(table_list):
                 table_no = int(user_input)
-                return table_list[table_no - 1]
+                table = table_list[table_no - 1]
+                return table
```
Comment (Cadair) [inline sunpy/net/helio/hec.py:192]: Why?

Comment (nabobalis) [reply sunpy/net/helio/hec.py:192]: I was working on something else and undid it but forgot to undo this.

File: sunpy/net/helio/hec.py [hunk 3]
```diff
@@ -174,14 +180,14 @@ def select_table(self):
...
```
Comment (nabobalis) [inline sunpy/net/helio/hec.py:191]:
```suggestion
                return table_list[table_no - 1]
```

**Comment by nabobalis** [comment]
**SQUASH MERGE**

**Review by Cadair — APPROVED** [review]

**Pull request merged**